get_velocity.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
+
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from collections import OrderedDict
+from datetime import timedelta, datetime
+from time import sleep
+from typing import List, Tuple, Optional
+
+import click
+from github import GithubException
+from ogr import GithubService
+from ogr.abstract import IssueStatus
+from ogr.services.github import GithubProject, GithubIssue
+
+
+def get_closed_issues(
+    project: GithubProject, tries=10, interval=10
+) -> List[GithubIssue]:
+    last_ex: Optional[GithubException] = None
+    for _ in range(tries):
+        try:
+            return project.get_issue_list(status=IssueStatus.closed)  # type: ignore
+        except GithubException as ex:
+            last_ex = ex
+            click.echo("Retrying because of the API limit.")
+            sleep(interval)
+    else:
+        raise last_ex  # type: ignore
+
+
+def is_counted(issue: GithubIssue) -> bool:
+    if issue._raw_issue.state_reason == "not_planned":
+        return False
+    if not issue.assignees:
+        return False
+    return True
+
+
+def get_week_representation(date_time) -> str:
+    year = date_time.year
+    week = date_time.isocalendar()[1]
+    week_str = f"{year}-{week:02d}"
+    return week_str
+
+
+def get_issue_value(
+    issue: GithubIssue,
+    labels: Optional[List[Tuple[str, int]]] = None,
+    value_without_label=0,
+):
+    if not labels:
+        return 1
+
+    issue_labels = [label.name for label in issue.labels]
+    value: Optional[int] = None
+    for label_name, label_value in labels:
+        if label_name in issue_labels:
+            value = label_value
+
+    if value is not None:
+        return value
```

Comment:
so this should work as `Later definition has priority` right? Maybe it would also make sense to always get the max value?

Reply:
> so this should work as Later definition has priority right? 

Yes, but `max` might be a better idea.